easybuild/tools/utilities.py

```diff
@@ -99,5 +102,9 @@ def import_available_modules(namespace):
                 mod_name = module.split(os.path.sep)[-1].split('.')[0]
                 modpath = '.'.join([namespace, mod_name])
                 _log.debug("importing module %s" % modpath)
-                modules.append(__import__(modpath, globals(), locals(), ['']))
+                try:
+                    mod = __import__(modpath, globals(), locals(), [''])
+                except ImportError as err:
```

Comment:
The general (unwritten) rule-of-thumb in EasyBuild is that errors that may be expected (e.g., caused by system issues) don't yield a traceback. Problems that do yield a traceback point to a bug in EasyBuild.

The functions we're discussing here are not intended to be of general use (at least not when including here), and are considered to be fatal when a problem occur caused by an external issue (e.g. filesystem, etc.).

For those type of issues, I want the user to get a clear error message (what was tried, what went wrong), not a full traceback (that's simply too daunting for end-users).

So: this idiom is applied all over the EB codebase.

If we want this changed, it should be i) discussed in a separate issue, ii) handled via a separate PR. I'm just sticking to the way of doing things here.